CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8_opt.c

```diff
@@ -22,7 +22,7 @@
  * Description:  Optimized s8 depthwise separable convolution function for
  *               channel multiplier of 1.
  *
- * $Date:        January 21, 2020
+ * $Date:        February 2020
```

Comment:
Looks good with the performance improvements on this. Lets make it as its own function(arm_depthwise_conv_3x3_s8()) to not mix the use of additional buffers. I can see it being used as it is for the MVE case with just just a modification for the requantization. 

When that is done, please add it to the TFLu glue as well under a defined(ARM_MATH_DSP) && !defined(ARM_METH_MVEI).

Reply:
Done. Will push to TFLu once this is merged